android/src/main/java/io/card/react/ReactCardIOModule.java

```diff
@@ -69,6 +69,11 @@ public void scan(ReadableMap options, Promise promise) {
 
       getReactApplicationContext().startActivityForResult(scanIntent, REQUEST_CARD_SCAN, null);
     }
+    
+    public void onActivityResult(android.app.Activity activity, int requestCode, int resultCode, Intent intent) {
+      Log.i("JAY","Jay Activity "+activity+" Result " + requestCode + " result Code " + resultCode);
```

Comment:
Please use something like this instead of `Jay` 😆 

```
Log.i(TAG,"Activity" + activity + " Result" + requestCode + " result Code " + resultCode);
```